tests/integration/cmd_push_test.go

```diff
@@ -210,7 +106,7 @@ var _ = Describe("odo push command tests", func() {
 			helper.CmdShouldPass("odo", "push", "--context", commonVar.Context, "-v4")
 
 			// Then check to see if it's truly been deleted
-			stdOut = oc.ExecListDir(podName, commonVar.Project, dir)
+			stdOut = oc.ExecListDir(podName, commonVar.Project, "/tmp/projects")
```

Comment:
Why are we hardcoding the dir part?

Reply:
Because it doesn’t change

Reply:
still made that use the constant from the convert.go now